libcloudforensics/scripts/ebs_snapshot_copy_aws.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash -x
+
+set -o pipefail
+
+snapshot=%s
+bucket=%s
+
+function ebsCopy {
+	# params
+	snapshot=$1
+	bucket=$2
+
+	# Get details about self
+	region=$(curl -s http://169.254.169.254/latest/meta-data/placement/region)
```

Comment:
What is this IP address? Is it guaranteed to be static?

Reply:
Yes, it is the instance metadata address